### DISCUSSION TITLE
How to do persistent storage?

### DISCUSSION THREAD

**Discussion by Jayceh2** [CosmosOS/Cosmos#2833] [Q&A]
Using the VFS works but when the OS is reboot the files are basically gone so I would assume they're only stored in memory?

**Comment by valentinbreiz** [comment]
Hello,
Are you using the VFS with a FAT32 formatted disk or with the ISO9660 filesystem?

**Reply by Jayceh2** [reply]
I'm honestly not quite sure since I'm just running it on default setup using a virtual system. Unless there's some configurations there that I can do? Or do I have to create a partition for it and run the virtual system there?
